### ISSUE TITLE
Feature Timeline wont load in Firefox 60.X.XX ESR

### ISSUE THREAD

**Issue by Christ0phH** [Microsoft/featuretimeline#79]
**Describe the bug**
When clicking on the feature timeline in Firefox, the browser switches the tab but loads the extension infinitely without ever displaying anything.

This occurs in our enterprise environment while using the special ESR version of Firefox. (Update to a newer release in not an option)

This is a reopen of the Issue #7 !

**System Info / Tested with**
* FireFox 60.4.0esr

* TFS 2018.2

* Feature Timeline Version 0.0.336
* Feature Timeline Version 0.0.338

**Browser log of Version 0.0.336 (identical) to 0.0.338**
```
uncaught at watchSagaActions at watchSagaActions 
 at takeEvery 
 at initializeFeatureState 
 getFeatureCookies@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:48805:13
initializeFeatureState/<@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:48784:39
step@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:48771:18
verb/<@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:48752:53
next@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:42016:18
proc@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:41975:3
runForkEffect@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:42259:19
runEffect@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:42140:800
next@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:42020:9
currCb@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:42093:7
takeCb@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:42171:143
put@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:41152:16
eventChannel/unsubscribe<@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:41240:5
stdChannel/chan</</<@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:41265:16
exec@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:42837:5
flush@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:42878:5
asap@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:42851:5
stdChannel/chan</<@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:41264:7
emit@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:41105:7
sagaMiddleware/</<@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:41668:9
./src/FeatureTimeline/react/Components/FeatureTimelineGrid.tsx/__WEBPACK_AMD_DEFINE_RESULT__</exports.PrimaryGrid@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:48703:9
yh@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:95:430
lg@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:120:88
mg@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:120:386
gc@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:127:202
vb@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:126:230
ub@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:126:65
zd@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:124:449
ra@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:123:319
Id@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:138:163
kc@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:138:401
Sa.prototype.render@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:194:410
mc/<@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:141:437
Hd@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:136:458
mc@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:141:372
render@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/react-dom.js:196:265
initialize@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/FeatureTimeline.js:48294:13
@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/featuretimeline.html:70:21
it/</<@https://our.tfsurl/tfs/_apis/public/gallery/publisher/ms-devlabs/extension/workitem-feature-timeline-extension/0.0.336/assetbyname/dist/libs/VSS.SDK.min.js:2:10364
t.prototype.spread/<@https://our.tfsurl/tfs/_public/_Bundling/Content?bundle=vss-bundle-ext-core-vD2dz332u2WjHkk6MHCQbahFBp9T549WGlB-V-UiAyCM=:27:9400
c@https://our.tfsurl/tfs/_public/_Bundling/Content?bundle=vss-bundle-ext-core-vD2dz332u2WjHkk6MHCQbahFBp9T549WGlB-V-UiAyCM=:27:8039
t.prototype.then/</<@https://our.tfsurl/tfs/_public/_Bundling/Content?bundle=vss-bundle-ext-core-vD2dz332u2WjHkk6MHCQbahFBp9T549WGlB-V-UiAyCM=:27:8305
t/u.promiseDispatch@https://our.tfsurl/tfs/_public/_Bundling/Content?bundle=vss-bundle-ext-core-vD2dz332u2WjHkk6MHCQbahFBp9T549WGlB-V-UiAyCM=:27:2872
r/s.promiseDispatch/<@https://our.tfsurl/tfs/_public/_Bundling/Content?bundle=vss-bundle-ext-core-vD2dz332u2WjHkk6MHCQbahFBp9T549WGlB-V-UiAyCM=:27:1857
t@https://our.tfsurl/tfs/_public/_Bundling/Content?bundle=vss-bundle-ext-core-vD2dz332u2WjHkk6MHCQbahFBp9T549WGlB-V-UiAyCM=:27:5777
```

**Comment by Christ0phH** [comment]
@navneet-g  Any chance you have a look into this?

**Comment by navneet-g** [comment]
I am not able to repro this issue at my end with Firefox 60.4.0esr.
Can you try with latest version of Firefox 65?
If it still repros we can schedule a live debugging session.

**Comment by Christ0phH** [comment]
Sadly, my company requires the current Firefox ESR version so even if i could test it with 65 it won't solve the problem. 

So a debug session would be the 'only' and neccessary solution.

**Comment by navneet-g** [comment]
Sure ... let me know what time suites you. Also do you see any errors in Network tab?

**Comment by Christ0phH** [comment]
> Sure ... let me know what time suites you. Also do you see any errors in Network tab?

How does next week look like? I'll try to fit in your schedule since mine is not that booked in that week.
How do we contact each other?

**Comment by navneet-g** [comment]
Please send me an email at navneetg@microsoft.com, I can send a meeting invite.

**Comment by navneet-g** [comment]
@Christ0phH did you send email to me? I have not got it yet.

**Comment by Christ0phH** [comment]
> @Christ0phH did you send email to me? I have not got it yet.

Just msg you.

**Comment by navneet-g** [comment]
Thanks @Christ0phH  for helping me debug the issue, I have release a new version, please try it out and let me know if you still see the error.

**Comment by Christ0phH** [comment]
> 
> 
> Thanks @Christ0phH for helping me debug the issue, I have release a new version, please try it out and let me know if you still see the error.

FIXED with 0.0.339 !

Your'e awesome !
